sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/implementation/notifications/signaling/CommunicationSignalingClient.java

```diff
@@ -226,30 +217,14 @@ private void start(Context context, ISkypetokenProvider skypetokenProvider, Stri
             return;
         }
 
-        CloudType cloudType = NotificationUtils.getUserCloudTypeFromSkypeToken(skypeUserToken);
-        String resourceLocation = NotificationUtils.decodeResourceLocationFromJwtToken(skypeUserToken);
-        boolean isEUDBCountry =
-            resourceLocation != null && countriesEUDB.contains(resourceLocation.toLowerCase(Locale.ROOT));
-        String trouterUrl;
-        String registrarUrl;
+        // Get trouterUrl from calling chat gateway
+        String apiVersion = "2024-09-01";
+        RealTimeNotificationConfig realTimeNotificationConfig = realtimeNotificationConfigClient.getTrouterSettings(skypeUserToken, serviceEndpoint, apiVersion);
 
-        switch (cloudType) {
-            case Dod:
-                trouterUrl = TROUTER_HOSTNAME_DOD;
-                registrarUrl = TROUTER_REGISTRATION_HOSTNAME_DOD;
-                break;
-
-            case Gcch:
-                trouterUrl = TROUTER_HOSTNAME_GCCH;
-                registrarUrl = TROUTER_REGISTRATION_HOSTNAME_GCCH;
-                break;
-
-            case Public:
-            default:
-                trouterUrl = isEUDBCountry ? TROUTER_HOSTNAME_EUDB : TROUTER_HOSTNAME;
-                registrarUrl = TROUTER_REGISTRATION_HOSTNAME;
-                break;
-        }
+        // Remove the "https://" prefix from the URLs
+        String trouterHostname = realTimeNotificationConfig.getTrouterServiceUrl().replace("https://", "");
+        String registrarBasePath = realTimeNotificationConfig.getRegistrarServiceUrl().replace("https://", "");
+        logger.info(String.format("Received config from service. Trouter Hostname: %s, Registrar Base Path: %s", trouterHostname, registrarBasePath));
```

Comment:
Maybe logs like these should be at the verbose level. I'll leave it up to you though.

Reply:
Changed to warn state

Reply:
The WARN level is higher in priority than INFO and should be used to let users know when things go wrong but not in a critical way (e.g. the app can still function or was able to recover). WE should use INFO for most purposes and VERBOSE/DEBUG for more granular debugging.